src/Computer/GlideComputerAirData.cpp

```diff
@@ -132,10 +132,9 @@ GlideComputerAirData::ProcessVertical(const MoreData &basic,
                         calculated);
   AverageClimbRate(basic, calculated);
 
-  if (calculated.circling)
+  if (calculated.circling) {
```

Comment:
Why add curly braces here? This obscures the commit diff.

Reply:
I like curles.